automol/const.py

```diff
@@ -57,23 +65,19 @@ def reverse(cls, value: str):
 
     @classmethod
     def is_reversible(cls, value: str) -> bool:
-        """Is this reaction class reversible?
+        """Is this reaction class reversible?.
```

Comment:
To be consistent with the standard formatting that the linter wants, let's replace questions like these with condition al statements:
```
"""Whether this reaction class is reversible.
```